src/project/manifest.rs

```diff
@@ -243,6 +244,43 @@ impl Manifest {
         Ok(())
     }
 
+    /// Remove the platform(s) from the project
```

Comment:
Could you add tests for this manifest modifications?

Reply:
I added tests for `set_description`, `set_version`, `{add|remove}_platforms` and `{add|remove}_channels`